js/package.json

```diff
@@ -27,6 +27,7 @@
         "build": "yarn clean && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
         "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"} ' | npx json > lib/esm/package.json",
         "test": "node --experimental-specifier-resolution=node --loader ts-node/esm test/index.ts",
+        "test:jest": "jest",
```

Comment:
This can just be the "test" script, the test/index.ts file needs to be replaced with proper tests anyway. We can just move the file and rename the script that runs it for now.